subsys/mpsl/fem/common/include/mpsl_fem_utils.h

```diff
@@ -7,9 +7,10 @@
 #ifndef MPSL_FEM_UTILS_H__
 #define MPSL_FEM_UTILS_H__
 
-#if defined(CONFIG_MPSL_FEM_PIN_FORWARDER)
-
 #include <stdint.h>
+#include <mpsl_fem_config_common.h>
```

Comment:
This file should only be included when `!defined(CONFIG_MPSL_FEM_PIN_FORWARDER)`. It can be achieved nicely with rearranging the file a bit:
```c
#include <stdint.h>
#if !defined(CONFIG_MPSL_FEM_PIN_FORWARDER)
#include <mpsl_fem_config_common.h>

void mpsl_fem_extended_pin_to_mpsl_fem_pin(uint8_t pin_num, mpsl_fem_pin_t *fem_pin);

#else /* !defined(CONFIG_MPSL_FEM_PIN_FORWARDER) */

void mpsl_fem_pin_extend_with_port(uint8_t *pin, const char *lbl);

#endif /* !defined(CONFIG_MPSL_FEM_PIN_FORWARDER) */
```

Reply:
Fixed